src/Components/MoodRating/MoodRating.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react'
```

Comment:
Thanks so much for thoroughly testing this new component!

Reply:
Same, I really appreciate y'all for your thoughtfulness/thoroughness in testing!